packages/block-library/src/gallery/edit.js

```diff
@@ -548,50 +542,49 @@ function GalleryEdit( props ) {
 							__next40pxDefaultSize
 						/>
 					) }
+					{ imageSizeOptions?.length > 0 && (
+						<SelectControl
+							__nextHasNoMarginBottom
+							label={ __( 'Resolution' ) }
+							help={ __(
+								'Select the size of the source images.'
+							) }
+							value={ sizeSlug }
+							options={ imageSizeOptions }
+							onChange={ updateImagesSize }
+							hideCancelButton={ true }
+							size="__unstable-large"
+						/>
+					) }
+					<SelectControl
+						__nextHasNoMarginBottom
+						label={ __( 'Link to' ) }
+						value={ linkTo }
+						onChange={ setLinkTo }
+						options={ linkOptions }
+						hideCancelButton={ true }
+						size="__unstable-large"
+					/>
 					<ToggleControl
 						__nextHasNoMarginBottom
 						label={ __( 'Crop images' ) }
 						checked={ !! imageCrop }
 						onChange={ toggleImageCrop }
-						help={ getImageCropHelp }
 					/>
 					<ToggleControl
 						__nextHasNoMarginBottom
-						label={ __( 'Random order' ) }
+						label={ __( 'Randomize images' ) }
```

Comment:
This seems slightly ambiguous now. It could be interpreted that the images themselves will be randomised rather than the order.

Reply:
Though I do think "order" isn't necessarily clear either. "Randomize image order" feels a bit technical/robotic almost. Other alternatives?

Reply:
I guess it's subjective but "Randomize order" seems pretty explicit to me.

Do you think there could be other ordering preset options in the future... Custom, Name, Date, Random... that might influence the UI?

Reply:
> Do you think there could be other ordering preset options in the future... Custom, Name, Date, Random... that might influence the UI?

I don't think so. 

> Randomize order

Let's roll with it. I'll update.

Reply:
> Randomize order

I just saw the screenshot below with this. I wasn't active in the last weeks and didn't know that this feature existed. Therefore, my initial thought when reading "Randomize order" was that the images will randomize on every page load. "Random order" sounds more like it's randomized only once aka more static.

When thinking about it, I wonder if a button wouldn't be a better component than a toggle. Toggle "Random order" and then having the button to "Randomize order".
Sorry, going a bit off topic to the original PR and I haven't tested the feature yet. But maybe it helps with the wording 🙂

Reply:
Ah! Just saw it actually randomize the order on every page load!
Well then "Randomize order" is better than "Random order". 
We could even add "Randomize order on every page load" or so to be even more clear.

Reply:
In the classic editor's gallery function, there is a checkbox called "Random Order". With this in mind, I think the toggle UI makes sense and the " on every page load" supplement is unnecessary.

![image](https://github.com/WordPress/gutenberg/assets/54422211/22ce646a-fe17-44b1-811d-3c7eadcb0d5a)

I don't have a strong opinion on whether "Random order" or "Randomize order" is better.

Reply:
> With this in mind, I think the toggle UI makes sense and the " on every page load" supplement is unnecessary.

People might expect a visual feedback in the editor when toggling. We prevent that by adding that it's only on page loads.

Reply:
As a side note for improving the UX: Every action should evoke a reaction. This could also be a message in with the notification bar: "The Gallery order will change on every page load" or so

Reply:
In this PR, I think that decisions should be made by focusing on two things: renaming and ordering, rather than the UX of the Gallery block. Also, if we need a broader discussion about naming, I think we can change the "Randomize order" changed in this PR back to the original "Random order".

@richtabor What do you think?

Reply:
> In this PR, I think that decisions should be made by focusing on two things: renaming and ordering, rather than the UX of the Gallery block.

I tend to agree but would like to add that naming/copy and UX are extremely intertwined. When the text states 'Randomize order on every page load', we would not need visual feedback because it's clear that the effect occurs on the page (and not in the editor) and only when the page is loaded. When it simply says 'Random order' or 'Randomize order', the user does not receive immediate feedback after toggling, leaving them unclear about what they have actually changed. There are many different ways to create feedback and indicate what, when, or where something has changed. One method could be through a notification bar, therefore the separate issue.

Reply:
> I think we can change the "Randomize order" changed in this PR back to the original "Random order".

To randomize something is much more clearer and action-oriented, than "to random" something. 

>  There are many different ways to create feedback and indicate what, when, or where something has changed. One method could be through a notification bar, therefore the separate issue.

I don't think we need a snackbar for this, but let's open an issue to discuss that apart. 🙇

Reply:
Ah, I see you did — nice! https://github.com/WordPress/gutenberg/issues/58558